src/entities/pools/gyroE/testingHelpers.ts

```diff
@@ -0,0 +1,101 @@
+import { parseUnits } from 'viem';
+import { DerivedGyroEParams, GyroEParams } from './gyroEPool';
+
+const ONE_100 = parseUnits('1', 100);
+
+export function calculateDerivedValues(p: GyroEParams): DerivedGyroEParams {
+    let { alpha, beta, s, c, lambda } = p;
+    [alpha, beta, s, c, lambda] = [alpha, beta, s, c, lambda].map((bn) =>
+        scale(bn, 18, 100),
```

Comment:
I couldn't find them on Gyro repo.
Considering they are test helpers maybe they don't exist on any solidity implementation?
Since it's used to setup a pool in order to run a unit test, I believe we could simply replace that with data from a real pool.
I'll go ahead and make that change 👍